packages/solana/package.json

```diff
@@ -0,0 +1,98 @@
+{
+  "name": "@web3modal/solana",
+  "version": "4.0.11",
+  "type": "module",
+  "main": "./dist/esm/exports/index.js",
+  "types": "./dist/types/exports/index.d.ts",
+  "files": [
+    "dist",
+    "!tsconfig.tsbuildinfo"
+  ],
+  "exports": {
+    ".": {
+      "types": "./dist/types/exports/index.d.ts",
+      "import": "./dist/esm/exports/index.js",
+      "default": "./dist/esm/exports/index.js"
+    },
+    "./react": {
+      "types": "./dist/types/exports/react.d.ts",
+      "import": "./dist/esm/exports/react.js",
+      "default": "./dist/esm/exports/react.js"
+    }
+  },
+  "typesVersions": {
+    "*": {
+      "react": [
+        "./dist/types/exports/react.d.ts"
+      ]
+    }
+  },
+  "scripts": {
+    "build:clean": "rm -rf dist",
+    "build": "tsc --build",
+    "watch": "tsc --watch",
+    "typecheck": "tsc --noEmit",
+    "lint": "eslint . --ext .js,.jsx,.ts,.tsx"
+  },
+  "dependencies": {
+    "@solana/wallet-adapter-backpack": "0.1.14",
+    "@solana/wallet-adapter-base": "0.9.23",
+    "@solana/wallet-adapter-phantom": "0.9.24",
+    "@solana/wallet-adapter-solflare": "0.6.28",
+    "@solana/wallet-adapter-trust": "0.1.13",
+    "@solana/wallet-adapter-walletconnect": "0.1.16",
+    "@web3modal/polyfills": "4.0.11",
+    "@web3modal/scaffold": "4.0.11",
+    "@web3modal/scaffold-react": "4.0.11",
+    "@web3modal/scaffold-utils": "4.0.11",
+    "@web3modal/scaffold-vue": "4.0.11",
+    "bn.js": "5.2.1",
+    "bs58": "5.0.0"
+  },
+  "devDependencies": {
+    "@types/bn.js": "5.1.5",
+    "@web3modal/siwe": "*"
+  },
+  "peerDependencies": {
+    "@solana/web3.js": ">=1.8",
+    "@web3modal/siwe": "*",
+    "react": ">=17",
+    "react-dom": ">=17",
+    "viem": ">=1",
+    "vue": ">=3"
+  },
+  "peerDependenciesMeta": {
+    "react": {
+      "optional": true
+    },
+    "react-dom": {
+      "optional": true
+    },
+    "@web3modal/siwe": {
+      "optional": true
+    }
+  },
```

Comment:
@glitch-txs I think we could remove these correct?

Reply:
yes, SIWE should be removed